js/app.js

```diff
@@ -11,10 +11,15 @@ const today = new Date();
 today.setUTCHours(0, 0, 0, 0);
 
 let seed = today.getTime();
+let alertDiv = document.getElementById("alert-box");
+alertDiv.style.visibility = "hidden";
+
 const url = new URL(document.location);
 const urlSeed = url.searchParams.get("seed");
+let newUrl = "";
```

Comment:
Is there a reason that this `newUrl` variable is defined here? It seems to be set (again) to an empty string on line 22, however it is set and immediately used inside the `generate-seed` click event listener anyway.